subscription-service/src/main/kotlin/com/egm/stellio/subscription/service/SubscriptionService.kt

```diff
@@ -521,13 +539,15 @@ class SubscriptionService(
                    entity_selector.id as entity_id, entity_selector.id_pattern as id_pattern, 
                    entity_selector.type_selection as type_selection, georel, geometry, coordinates, pgis_geometry,
                    geoproperty, scope_q, notif_attributes, notif_format, endpoint_uri, endpoint_accept, times_sent, 
-                   endpoint_receiver_info, endpoint_notifier_info, contexts
+                   endpoint_receiver_info, endpoint_notifier_info, contexts, throttling
             FROM subscription 
             LEFT JOIN entity_selector on subscription.id = entity_selector.subscription_id
             LEFT JOIN geometry_query on subscription.id = geometry_query.subscription_id
             WHERE is_active
             AND ( expires_at is null OR expires_at >= :date )
             AND time_interval IS NULL
+            AND ( throttling IS NULL 
+                OR (last_notification + throttling * INTERVAL '1 second') > CURRENT_TIMESTAMP )
```

Comment:
Dates in NGSI-LD are always on UTC timezone. `CURRENT_TIMESTAMP` will have the local timezone, not sure how PostgreSQL will handle the comparison